src/thunderbots/shared/constants.h

```diff
@@ -12,6 +12,9 @@ const double BALL_MAX_SPEED_METERS_PER_SECOND = 6.5;
 const double ROBOT_MAX_RADIUS_METERS = 0.09;
 // The distance from the center of the robot to the front face (the flat part), in meters
 const double DIST_TO_FRONT_OF_ROBOT_METERS = 0.07;
+// The distance from one end of the dribbler to the other
+// TODO: actually measure this value
+const double DRIBBLER_LENGTH = 0.05;
```

Comment:
I think using the dribbler length is a good way to determine an area range in which the ball is considered to be "in possession". If you guys agree I will actually measure this value in the next few days.

Reply:
I happened to be in the Bay while looking at this and measured the values.
* The actual dribbler component has a total width of 0.088m (8.8cm)
* The entire width of the flat front of the robot is 0.11m (11cm)

In general I agree this is a reasonable value to use to base the region off of.

Reply:
Awesome. I'll add it to the constants as part of this PR.